app/src/main/java/com/mapbox/dash/example/MainApplication.kt

```diff
@@ -62,6 +64,11 @@ class MainApplication : Application() {
                 resultsAdapter = ShowcaseSearchResultsAdapter()
                 searchEngine = ShowcaseSearchEngine()
             }
+            offline {
+                tilesPath = applicationContext.filesDir.absolutePath + "/custom_tiles_path"
+                tilesDiskQuota = 15.Gb
+                fallbackNavigationTilesVersion = "2024_06_23-03_00_01"
```

Comment:
@VysotskiVadim should we still specify tiles version explicitly or it might be obtained via style's metadata?